ni/protobuf/types/waveform_wrappers.proto

```diff
@@ -0,0 +1,42 @@
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+syntax = "proto3";
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+package ni.protobuf.types;
+
+import "ni/protobuf/types/waveform.proto";
+
+//---------------------------------------------------------------------
+//---------------------------------------------------------------------
+option csharp_namespace = "NationalInstruments.Protobuf.Types";
+option go_package = "types";
+option java_multiple_files = true;
+option java_outer_classname = "WaveformWrappersProto";
+option java_package = "com.ni.protobuf.types";
+option objc_class_prefix = "NIPT";
+option php_namespace = "NI\\PROTOBUF\\TYPES";
+option ruby_package = "NI::Protobuf::Types";
+
+// These shall only be used with oneof fields and when packing into an Any message
```

Comment:
Borrowing from style from [wrappers.proto](https://github.com/protobuf-net/protobuf-net/blob/main/src/protobuf-net.Reflection/google/protobuf/wrappers.proto):

// Wrappers for common waveform message types. These types are useful
// for embedding in the `google.protobuf.Any` type or in oneof fields.
// Their use outside of these scenarios is discouraged.